logicrunner/artifacts/client.go

```diff
@@ -354,12 +354,12 @@ func (m *client) GetObject(
 	return desc, err
 }
 
-func (m *client) GetIncomingRequest(
+func (m *client) GetIncomingOrOutgoingRequest(
 	ctx context.Context, object, reqRef insolar.Reference,
-) (*record.IncomingRequest, error) {
+) (*record.IncomingRequest, *record.OutgoingRequest, error) {
```

Comment:
seems strange, always returns only one of Request. Need to use interfaces or something.

Reply:
ack